### DISCUSSION TITLE
ChromiumOxidation and ChromiumThermal is not a registered object

### DISCUSSION THREAD

**Discussion by haqfariha** [idaholab/moose#24149] [Q&A Getting Started] [answered]
I am a new user and trying to work with oxidation problem for my project. I am trying built-in test input file of moose but I can not run it because of error messeges pop up - these kernels are not registered. How can I resolve this? The problem I am trying-
https://mooseframework.inl.gov/bison/source/materials/ChromiumOxidation.html#chromiumoxidation
Thanks in advance.

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is actually from the BISON website, and these are BISON objects.
BISON is an export controlled application application, you may request access here:

https://ncrcaims.inl.gov/Identity/Account/Register

Guillaume
